### PR TITLE
Add more Suspension Properties

### PR DESCRIPTION
**Pull request by jgersti**
Adds attached as a readonly property. (#174)

We should take a look what other properties can be easily incorporated in this PR.

### CODE REVIEW

File: docs/EasyCommands/blockHandlers/suspension.md [hunk 1]
```diff
@@ -156,4 +156,16 @@ Print "Wheel Friction: " + "My Wheel" ratio
...
+
+## "Attached" Property
+* Primitive Type: Bool
+* Keywords: ```attached, connected```
+* Inverse Keywords: ```detached, disconnected```
+
+Returns true if there is a wheel attached to the suspension.
+
+```
+if "My Wheel" attached
```
Comment (MerlinofMines) [inline docs/EasyCommands/blockHandlers/suspension.md:169]: nit: if "My Wheel" is attached

File: docs/EasyCommands/blockHandlers/suspension.md [hunk 2]
```diff
@@ -156,4 +156,16 @@ Print "Wheel Friction: " + "My Wheel" ratio
...
+  Print "Something stole the Wheel!"
```
Comment (MerlinofMines) [inline docs/EasyCommands/blockHandlers/suspension.md:170]: haha love it :)

**Comment by MerlinofMines** [comment]
See some of my comments on #176  and #175 for thoughts on expanding the PR to support those properties.  Adding to this PR or doing as a separate PR both seem fine.

File: docs/EasyCommands/blockHandlers/suspension.md [hunk 3]
```diff
@@ -156,4 +156,16 @@ Print "Wheel Friction: " + "My Wheel" ratio
...
+
+## "Attached" Property
+* Primitive Type: Bool
+* Keywords: ```attached, connected```
+* Inverse Keywords: ```detached, disconnected```
+
+Returns true if there is a wheel attached to the suspension.
+
+```
+if "My Wheel" is attached
```
Comment (MerlinofMines) [inline docs/EasyCommands/blockHandlers/suspension.md:169]: i think you mean if "My Wheel" is detached, or not attached.

Comment (jgersti) [reply docs/EasyCommands/blockHandlers/suspension.md:169]: ups

**Comment by MerlinofMines** [comment]
Do you want to merge as is or keep iterating on the PR?

**Comment by jgersti** [comment]
merge it.
the other stuff needs some more thoughts (and time)

**Comment by MerlinofMines** [comment]
Happy to merge but curious what you think on adding attach/detach support..wouldn't be difficult.


 AddBooleanHandler(Property.CONNECTED, b => b.IsAttached, (b,v) => {if(v) b.Attach(); else b.Detach(); });

File: EasyCommands/BlockHandlers/WheelBlockHandlers.cs [hunk 4]
```diff
@@ -23,7 +23,8 @@ public class WheelSuspensionBlockHandler : FunctionalBlockHandler<IMyMotorSuspen
             public WheelSuspensionBlockHandler() {
                 AddNumericHandler(Property.LEVEL, b => b.Height, (b, v) => b.Height = v, 0.1f);
                 AddNumericHandler(Property.ANGLE, b => (float)(-b.SteerAngle*180/Math.PI), (b, v) => { b.SteeringOverride = (float)(v * Math.PI / 144); b.MaxSteerAngle = 1; }, .1f);
-                AddBooleanHandler(Property.LOCKED, (b) => b.Brake, (b, v) => b.Brake = v);
+                AddBooleanHandler(Property.LOCKED, b => b.Brake, (b, v) => b.Brake = v);
+                AddBooleanHandler(Property.CONNECTED, b => b.IsAttached);
```
[the remark this answers is not in the document]
Comment (jgersti) [reply EasyCommands/BlockHandlers/WheelBlockHandlers.cs:27]: I am not sure you can actually attach anything but a rotor head this way. I need to test that ingame later.

Adding a new wheel is hidden behind an action on the suspension and silently fails if there is no space to put the wheel.

[the remark this answers is not in the document]
Comment (MerlinofMines) [reply EasyCommands/BlockHandlers/WheelBlockHandlers.cs:27]: Yep, it'd be best effort, similar to connecting a connector

```
while "My Wheel" is not attached
  attach "My Wheel"
```

I'll add a different issue to be investigated for this

**Review by MerlinofMines — APPROVED** [review]

**Pull request merged**